src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="conteiner">
+      <div class="container__star stars--0">
+        <div class="star"></div>
+        <div class="star"></div>
+        <div class="star"></div>
+        <div class="star"></div>
+        <div class="star"></div>
+      </div>
+      <div class="container__star stars--1">
```

Comment:
checklist point 1 (but do not leave empty lines if the tag contains only 1 line)

```suggestion
      </div>
   
      <div class="container__star stars--1">
```